app/assets/config/manifest.js

```diff
@@ -1,2 +1,2 @@
 //= link_tree ../images
-//= link application.css
+//= link_directory ../stylesheets .css
```

Comment:
`manifest.js` is a sprockets thing, so should we delete it?

Reply:
Glad you brought this back up. I had tried originally and was getting `Expected to find a manifest file in `app/assets/config/manifest.js` (Sprockets::Railtie::ManifestNeededError)`, but I think enough got deleted that the app will boot without anything in `app/assets` now.